src/client/Microsoft.Identity.Client/Internal/Logger/NullLogger.cs

```diff
@@ -21,7 +21,7 @@ internal class NullLogger : ILoggerAdapter
 
         public bool IsDefaultPlatformLoggingEnabled { get; } = false;
 
-        public IIdentityLogger IdentityLogger { get; } = null;
+        public IIdentityLogger IdentityLogger { get; } = new NullIdentityLogger();
```

Comment:
nit: Can a NullIdentityLogger from the IdentityModel abstractions package be used here?

Reply:
Yes, didnt realize it existed.